new-lamassu-admin/src/pages/Locales/Locales.js

```diff
@@ -10,7 +10,9 @@ import { Table as EditableTable } from 'src/components/editableTable'
 import Section from 'src/components/layout/Section'
 import TitleSection from 'src/components/layout/TitleSection'
 import { P } from 'src/components/typography'
-import { fromNamespace, toNamespace } from 'src/utils/config'
+import { fromNamespace, toNamespace, namespaces } from 'src/utils/config'
+
+import Wizard from '../Wallet/Wizard'
```

Comment:
I would rather we never have to use `../`, since this is a import from something outside of the module we should make them full named imports like the ones above.